src/App.js

```diff
@@ -1,23 +1,76 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import htmlContent from "./content.json";
+import styles from "./App.module.scss";
+import Header from "../src/component/header/header";
+import Footer from "../src/component/footer/footer";
+import Main from "../src/component/main/main";
 
```

Comment:
Try to use Typescript in your whole project:
https://create-react-app.dev/docs/adding-typescript/
https://www.sitepoint.com/react-with-typescript-best-practices/
https://blog.logrocket.com/using-typescript-with-react-tutorial-examples/